design/002-Package-Structure-Code-Organiation.md

```diff
@@ -0,0 +1,90 @@
+<!-- This is a template for proposing design changes to the dst-go project. -->
+
+# Proposal: <!-- Package structure and code organization for dst-go -->
```

Comment:
The proposal title should not be commented out.
```suggestion
# Proposal: Package structure and code organization for dst-go
```

Reply:
Updated the changes in this commit - fafe44f.